base/applications/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_subdirectory(cacls)
 add_subdirectory(calc)
 add_subdirectory(charmap)
 add_subdirectory(clipbrd)
+add_subdirectory(cleanmgr)
```

Comment:
The rest of this list seems sorted alphabetically, so might be nice to do that for this one as well.